src/components/Faq.content.tsx

```diff
@@ -5,8 +5,9 @@ export const faqQuestionsAnswers = [
 		answer: (
 			<>
 				<p>
-					A little bit of money can go a long way in data storage. For $1 USD you can purchase storage for
-					approximately 1000 documents, 55 photos, or a 30 second video.
+					A little bit of money can go a long way in data storage. As you can see with the price calculator
+					for a small amount of USD you can purchase storage for hundreds of documents or dozens of photos or
+					songs.
```

Comment:
I'm feeling two 'or' pretty close there. Does it worth changing the last one for an 'and'?
I'm not THAT good at English, it might be ok as it is :sweat_smile: 

```suggestion
					for a small amount of USD you can purchase storage for hundreds of documents or dozens of photos and
					songs.
```

Reply:
Yeah, I think that would improve the grammar a bit. But the requirements of the ticket right now are to match Wayne's update on the answer here: https://ardrive.atlassian.net/wiki/spaces/ENGINEERIN/pages/146997264/ArDrive+Price+Calculator+Questions+and+Answers